website/content/terminal/usage/basics/index.md

```diff
@@ -1,114 +1,294 @@
+```console
+support -h
+```
 
-Let's change the starting and ending period of the data that is being loaded in by doing the following:
+```console
+Submit your support request
 
+options:
+  -c {search,load,quote,tob,candle,news,resources,codes,ta,ba,qa,disc,dps,scr,sia,ins,gov,res,dd,fa,bt,ca,options,th,forecast}, --command {generic,search,load,quote,tob,candle,news,resources,codes,ta,ba,qa,disc,dps,scr,sia,ins,gov,res,dd,fa,bt,ca,options,th,forecast}
+                        Command that needs support (default: None)
+  --msg MSG [MSG ...], -m MSG [MSG ...]
+                        Message to send. Enclose it with double quotes (default: )
+  --type {bug,suggestion,question,generic}, -t {bug,suggestion,question,generic}
+                        Support ticket type (default: generic)
+  -h, --help            show this help message (default: False)
 ```
-(🦋) /stocks/ $ load AMZN -s 2005-01-01 -e 2010-01-01
 
-Loading Daily data for AMZN with starting period 2005-01-03.
+![Support](https://user-images.githubusercontent.com/85772166/233577183-fbeb7be2-1d00-4ca0-86b3-42f1b71081e8.png)
 
+```console
+support search --type question --msg "How do I find stocks from India with OpenBB?"
 ```
 
-We can check that this period has changed by looking into the <a href="https://www.investopedia.com/trading/candlestick-charting-what-is-it/" target="_blank" rel="noreferrer noopener">candle chart</a> with `candle`. This, again shares the same `-h` argument. This results in the following which indeed depicts our selected period.
+![Support Auto Complete](https://user-images.githubusercontent.com/85772166/233577389-f0ad1b08-0e22-44b1-9d6b-9732c77af7d7.png)
+
+The command opens a browser window to a pre-populated form on the OpenBB website.  If you are signed-in to the Hub, all that is left to do is click `Submit`.
+
+![Submit Form](https://user-images.githubusercontent.com/85772166/233577448-3e426a88-d0cf-4338-8f4c-21b9fd01d8b2.png)
 
+:::note
+An answer to this question is: `search --country india --exchange-country india`
+:::
+
+:::note
+Tips for submitting a support request:
+
+- Tell us what version number is installed.
+- Tell us what operating system and version the machine has.
+- What is the installation type?  Installer, Source, PyPi, Docker, other?
+- Tell us the command and parameter combination causing the error.
+- Tell us what symbol (ticker) is, or was trying to be, loaded.
+- Show us the complete error message.
+- Let us know any contextual information that will help us replicate and accurately identify the problem.
+  :::
+
```

Comment:
for some reason this doesnt end the note:

<img width="704" alt="Screenshot 2023-04-21 at 3 57 03 PM" src="https://user-images.githubusercontent.com/18151143/233723986-7c089f2a-2b43-4457-8d26-0602a4c43fe0.png">


It also seems weird to have the two notes back to back.

Reply:
The last `:::` had managed to indent itself, so corrected that and made the first item a **bold statement**.